src/base/bittorrent/torrentimpl.cpp

```diff
@@ -1516,6 +1516,12 @@ qlonglong TorrentImpl::nextAnnounce() const
     return lt::total_seconds(m_nativeStatus.next_announce);
 }
 
+qreal TorrentImpl::interestRate() const
+{
+    qlonglong elapsed = activeTime();
+    return elapsed ? realRatio() / (elapsed / INTEREST_RATE_PERIOD) : 0;
```

Comment:
We discourage to use numbers as booleans.

Reply:
Can't `elapsed` ever be zero?

Reply:
> We discourage to use numbers as booleans.

A comparison to zero has been added.

> Can't `elapsed` ever be zero?

Yes, it's zero in case a torrent is added but not yet started due to the `Start torrent` checkbox was toggled.

Reply:
>Can't elapsed ever be zero?

Oops, i was asking for division by zero but didn't pay attention to the left side.